crates/humanode-rpc/src/lib.rs

```diff
@@ -295,7 +314,9 @@ where
             eth_fee_history_cache,
             eth_fee_history_limit,
             eth_execute_gas_limit_multiplier,
+            eth_forced_parent_hashes,
         )
+        .replace_config::<EC>()
```

Comment:
Why is this requied?

Reply:
The changes are based on - https://github.com/paritytech/frontier/commit/2b09a676cac8cca665c882a28bd13b2acef39395#diff-10dbae62d0fd1bd075ca0e84312de153926ca883ef247536affd3b6e1a9d9429.

Yes, we can directly use default `EthConfig`. Proposed changes are based on frontier template version.

Reply:
I think I know why the did this - to avoid having super long `<_,_,_...>` thing, let's keep it